SConstruct

```diff
@@ -1335,8 +1335,8 @@ if sanitizers:
         env.Die("sanitizers are not supported for compiler '%s'" % compiler)
 
     for name in sanitizers:
-        flags = ['-fsanitize=%s' % name]
-
+        flags = ['-fsanitize=%s' % name, '-fno-sanitize-recover=%s' % name, '-fsanitize-blacklist=blacklist.txt']
```

Comment:
Let's specify sanitizer blacklist option only once instead of repeating it.

BTW, is repeating sanitize-recover option with different values okay, or should we pass it once with a list?

Reply:
> Let's specify the sanitizer blacklist option only once instead of repeating it.
Yes, it was by mistake. I fixed it.

> BTW, is repeating sanitize-recover option with different values okay, or should we pass it once with a list?
From the documentation, it seems good to provide all options in a list. So I fixed it.